### ISSUE TITLE
Analysis results not appearing

### ISSUE THREAD

**Issue by csobier** [CartoDB/cartodb#13281]
@urbanphes, ever since the Geocode workflow change, the Analysis results screen is not appearing after running the the Geocode analysis? I'm noticing this for all Geocode TYPES selected.

FYI: The Analysis results _still appear_ correctly after other analysis (not Geocode) are run. However, there is some weirdness where I won't see the analysis results if I re-run an analysis? 

Can you define the expected Analysis Results behavior? This impacts several use cases in Learn.

**Comment by matallo** [comment]
ahoy @csobier ! from now on @alonsogarciapablo will take care of this discussions, so as much background info on the topic you could provide it will be highly appreciated 🙏

**Comment by alonsogarciapablo** [comment]
I guess we're referring to this overlay with info about the result of the analysis:

![screen shot 2017-12-28 at 15 20 39](https://user-images.githubusercontent.com/390398/34413296-bd1829e4-ebe2-11e7-9ddc-e5c903aecc79.png)

I'm missing some context here but I talked to @matallo and he explained to me that right now that overlay is only displayed the first time a given analysis finishes.

**Comment by csobier** [comment]
@alonsogarciapablo, OK, thanks. I understand that analysis results only appear once per map now but this issue is specific to the analysis results not appearing AT ALL when running a GEOCODE analysis (for all geocode types). 

I've tried for several different maps using all the geocode types, it never appears. Can you confirm? 
(I trying to see the results at least once for ensuring the terminology changes were applied, and for vetting legacy use cases when using the STYLE THIS ANALYSIS feature of the results)

**Comment by alonsogarciapablo** [comment]
From what I can see, it looks like we only display the results of each analysis the first time the user runs the analysis (for every map). This means that if you already ran a GEOCODE analysis with that user, you won't see the analysis results again.

This is stored in the browser's Local Storage so you might see the results in other computers (and perhaps browser sessions?):

<img width="876" alt="screen shot 2018-01-10 at 18 50 45" src="https://user-images.githubusercontent.com/390398/34787331-310f61b0-f637-11e7-94a2-09f20c8e5ca9.png">

**Comment by csobier** [comment]
Oh, so only once, ever. Hmmm, ok.  I will work with that behavior then. Thanks for clarifying.

**Comment by alonsogarciapablo** [comment]
This is how it works but I wasn't part of this project so I'm not sure if this is the expected behaviour.
@piensaenpixel can you clarify if this is how this should work (I'm not sure who designed this)? Thanks!

**Comment by piensaenpixel** [comment]
Uhmm as far as i know the message should appear every time that it finishes

**Comment by alonsogarciapablo** [comment]
@matallo it this something that Bravo implemented? could you please confirm if [this](https://github.com/CartoDB/cartodb/issues/13281#issuecomment-356682249) is a feature or a bug 🙏 ? Thanks!

**Comment by rubenmoya** [comment]
Sorry to jump in! 

I'm pretty sure this is related to https://github.com/CartoDB/cartodb/issues/13003, this is the PR related to that issue: https://github.com/CartoDB/cartodb/issues/13003

**Comment by matallo** [comment]
It appears, just the first time though, as per the changes in the issue @rubenmoya shared

so, if we want it ALWAYS to appear a new issue would have to be created (and prioritized via Response Team, or squad)

**Comment by piensaenpixel** [comment]
But ari's ticket is related to the on-boarding not to the analysis feedback they are different things.

**Comment by alonsogarciapablo** [comment]
I think we should revert this change then (#13039). The analysis results screen is important because it includes info about new columns, etc.

@ramiroaznar could prioritise this in the RT 🙏 ?

I'll open another issue to re-implement this (only for the onboarding modal) and add it to the Bravo kanban.

**Comment by matallo** [comment]
@piensaenpixel Well, they aren't different things actually, they share the same component, thus they behave the same

thanks @alonsogarciapablo !

**Comment by ramiroaznar** [comment]
Roger that!

**Comment by alonsogarciapablo** [comment]
Here's the new issue to re-implement this: https://github.com/CartoDB/cartodb/issues/13415. Vamos!

**Comment by piensaenpixel** [comment]
but i don't think we shouldn't revert it, why don't open an issue for fixing it?

**Comment by alonsogarciapablo** [comment]
We should revert it first (this should be easy so RT team can take care of it) so that it doesn't impact users and then Bravo can re-implement it properly (this might take a bit more time).

**Comment by piensaenpixel** [comment]
We are going to work two times, but disagree and commit :)

**Comment by ramiroaznar** [comment]
This is already fixed.
